shared/src/model/exercise/tag.js

```diff
@@ -54,12 +54,26 @@ class ExerciseTag extends BaseModel {
     return this.asString;
   }
 
+  @computed get sortValue() {
+    // APLO should always appear first
+    if ('aplo' === this.type) { return 'A'; }
+    // and the LO
+    if ('lo' === this.type) { return 'AA'; }
```

Comment:
Shouldn't we just prepend the A or AA in case there are more LO's?

Reply:
we also sort by the tag value itself, this is just to force the APLO & LO's to appear first.  
https://github.com/openstax/tutor-js/pull/2632/files#diff-7521d64a6f3a3194ea1a6b6c894ccbe5R84